quupe-RN/js/screens/SignIn/SignInContainer.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+import SignIn from './SignIn';
+
+class SignInContainer extends Component {
+  constructor() {
```

Comment:
currently, you don't have a state. You do not need a constructor. Please, remove it.